txtl-liposome_water-in-oil.md

```diff
@@ -0,0 +1,217 @@
+# TX-TL Liposome Using Water-in-Oil Emulsion
+
+Version 0.1.0, 8 August 2018
+
+Contributors:
+Aaron Engelhart (University of Minnesota),
+Jan Gregrowicz (Caltech),
+Joseph Heili (University of Minnesota),
+Zoila Jurado (Caltech),
+Neha Kama (Northwestern),
+Akshay Maheshwari (Stanford),
+Richard Murray (Caltech),
+Milena Popovic (Blue Marble Space),
+Kazhito Tabata (University of Tokyo),
+Paola Torre (University of Pennsylvania).
+
+## Overview and Materials
+
+This protocol describes how to create liposomes that contain TX-TL
+inside of a lipid-based container. The liposomes created by these
+protocols are between 1 and 10 um in diameter and should be usable
+with an cell-free protein expression system.
+
+The preparation of liposomes using w/o emulsions as template requires
+four steps:
+
+### **Step 0**: Deposition of a thin lipid film in a glass vial
+
+#### Materials:
+
+* Glass vials (any will do, but we use [2mL Fisherbrand Class B Clear
+  Glass Threaded vials: cat #:
+  03-339-21A](https://www.fishersci.com/shop/products/fisherbrand-class-b-clear-glass-threaded-vials-with-closures-packaged-separately-12/p-204738](https://www.fishersci.com/shop/products/fisherbrand-class-b-clear-glass-threaded-vials-with-closures-packaged-separately-12/p-204738)
+
+* POPC in chloroform: [https://avantilipids.com/product/850457](https://avantilipids.com/product/850457) 
+
+* Lyss-Rhod-PE:
```

Comment:
Should be Liss-Rhod-PE, per https://buildacell.slack.com/archives/C69RKEY3U/p1533790677000173.

Reply:
Here some details, maybe useful:
Sold from Avanti Polar Lipids, Inc., https://avantilipids.com/product/810150